src/main/java/com/dabsquared/gitlabjenkins/GitLabPushTrigger.java

```diff
@@ -551,9 +551,9 @@ protected Object readResolve() throws ObjectStreamException {
     public static GitLabPushTrigger getFromJob(Job<?, ?> job) {
         GitLabPushTrigger trigger = null;
         if (job instanceof ParameterizedJobMixIn.ParameterizedJob) {
-            ParameterizedJobMixIn.ParameterizedJob p = (ParameterizedJobMixIn.ParameterizedJob) job;
-            Collection<Trigger> triggerList = p.getTriggers().values();
-            for (Trigger t : triggerList) {
+            ParameterizedJobMixIn.ParameterizedJob<?, ?> p = (ParameterizedJobMixIn.ParameterizedJob) job;
+            Collection<Trigger<?>> triggerList = p.getTriggers().values();
```

Comment:
(subsuming #1358)

Reply:
I think that I will merge #1358 once it completes its CI run so that the changelog includes the text from #1358.  I merged it into this pull request to assure that this pull request would also verify.

Reply:
FYI, topologically (not squash- or rebase-) merging this PR would cause #1358 to be considered merged as well, I think according to Release Drafter too.

Reply:
Basically, a PR is “merged” when _either_
* it was squash- or rebase-merged by GitHub; or
* its head commit becomes an ancestor of its base branch, by any means (including e.g. `git merge` without any GH-specific tools).